.ci.yaml

```diff
@@ -167,6 +167,14 @@ targets:
       # ensure files from pre-production Fuchsia SDK tests are purged from cache
       clobber: "true"
     timeout: 90
+    runIfNot:
```

Comment:
This can also go on the engine_v2 version `Linux linux_fuchsia` down on line 225.

Reply:
Good catch! Re-ordered and added.